src/app/app.component.scss

```diff
@@ -84,3 +122,5 @@
     background-color: #007aa9;
   }
 }
+
+//If screen size is small
```

Comment:
Is this comment here deliberately? Looking further up, I see you're "closing off" the block of code. Since you're starting each block with a comment, I'm not sure a closing off is needed as with the comment at the start would indicate that the previous block is finished?

Reply:
Yeah thats fair, didnt think of it this was, i think i labeled them working from the bottom up so doing it it way i must not have noticed :p